src/core/query-builder.constants.ts

```diff
@@ -0,0 +1,201 @@
+import { QueryBuilderCustomOperation } from "smart-webcomponents-react";
+
+export const queryBuilderMessages = {
+  en: {
+    propertyUnknownType: "'' property is with undefined 'type' member!",
+    propertyInvalidValue: "Invalid '!",
+    propertyInvalidValueType: "Invalid '!",
+    elementNotInDOM: 'Element does not exist in DOM! Please, add the element to the DOM, before invoking a method.',
+    moduleUndefined: 'Module is undefined.',
+    missingReference: '.',
+    htmlTemplateNotSuported: ": Browser doesn't support HTMLTemplate elements.",
+    invalidTemplate: "' property accepts a string that must match the id of an HTMLTemplate element from the DOM.",
+    add: 'Add',
+    addCondition: 'Add Condition',
+    addGroup: 'Add Group',
+    and: 'And',
+    notand: 'Not And',
+    or: 'Or',
+    notor: 'Not Or',
+    '=': 'Equals',
+    '<>': 'Does not equal',
+    '>': 'Greater than',
+    '>=': 'Greater than or equal to',
+    '<': 'Less than',
+    '<=': 'Less than or equal to',
+    startswith: 'Starts with',
+    endswith: 'Ends with',
+    contains: 'Contains',
+    notcontains: 'Does not contain',
+    isblank: 'Is blank',
+    isnotblank: 'Is not blank',
+    wrongParentGroupIndex: "' method.",
+    missingFields:
+      ': Fields are required for proper condition\'s adding. Set "fields" source and then conditions will be added as expected.',
+    wrongElementNode: "' method.",
+    invalidDataStructure: ': Used invalid data structure in updateCondition/updateGroup method.',
+    dateTabLabel: 'DATE',
+    timeTabLabel: 'TIME',
+    queryLabel: '',
+  },
+};
+
+export const QueryBuilderOperations = {
+  EQUALS: {
+    label: 'Equals',
+    name: '=',
+    expressionTemplate: '{0} = "{1}"',
```

Comment:
You don't have to solve this now, not sure how this would work in case of numerics